### ISSUE TITLE
LEFTIST Output Is Independent of Data/Model

### ISSUE THREAD

**Issue by shrezaei** [fzi-forschungszentrum-informatik/TSInterpret#57]
**Describe the bug**
Regardless of the model and data, the output of the LEFTIST is either the first few time steps or last few time steps. In a few experiment I did, it seems that it purely depends on the class label, not the model or data. Even in the documentation (https://fzi-forschungszentrum-informatik.github.io/TSInterpret/Notebooks/Leftist_torch/) the same issue presents.
Here are a few examples where the distinguishable feature is the Sine wave segment of the time series.
![0](https://github.com/fzi-forschungszentrum-informatik/TSInterpret/assets/36204315/c1ab0e26-7474-4dd5-87bb-f909730e20fc)
![1](https://github.com/fzi-forschungszentrum-informatik/TSInterpret/assets/36204315/11062206-ecf0-4086-8c19-141a111e1de8)
![2](https://github.com/fzi-forschungszentrum-informatik/TSInterpret/assets/36204315/2cea3a00-5bc8-4d59-8113-350f23626fd2)
![3](https://github.com/fzi-forschungszentrum-informatik/TSInterpret/assets/36204315/5a0dd654-6164-4981-bdff-8e1f6ba0349f)

**To Reproduce**
Steps to reproduce the behavior. Also State the OS and the versions you are using.

**Additional context**

**Comment by JHoelli** [comment]
Hi @shrezaei,

Thanks for reporting the issue.

I found a small bug in the generation of the heatmaps from the explanations, it is fixed here:
`pip install https://github.com/fzi-forschungszentrum-informatik/TSInterpret/archive/refs/heads/LEFTIST.zip`.
(I will publish a new release when I am sure that everything is fixed.)

Other than that, I could not really find anything wrong. LEFTIST is unfortunately quite prone to the parameter settings used, so if you still have problems, providing yours would help me localize the issue. 

I also pushed a new notebook: https://github.com/fzi-forschungszentrum-informatik/TSInterpret/blob/LEFTIST/docs/Notebooks/Leftist_torch.ipynb

Hope that helps
